engine/access/access_test.go

```diff
@@ -110,6 +110,7 @@ func (suite *Suite) RunTest(
 			uint(9000),
 			nil,
 			false,
+			suite.log,
```

Comment:
I was actually going to do this by adding a [Payload Interceptor](https://github.com/grpc-ecosystem/go-grpc-middleware/blob/v2/interceptors/logging/payload.go) here https://github.com/onflow/flow-go/blob/9e3167ed415e45c27560bc0c6b9514757a5c2be9/engine/access/rpc/engine.go#L93. That would log the request payload (in debug).

But you beat me to it. You mind if I change the solution to use the interceptor once I have it working?